tests/test_unified_llm_runtime.py

```
@@ -0,0 +1,142 @@
+from types import SimpleNamespace
+
+import pytest
+import torch
+
+from monGARS.core.llm_integration import LLMRuntimeError, UnifiedLLMRuntime
+
+
+class FakeUnifiedLLMRuntime:
+    """Minimal stub used to document the runtime contract in unit tests."""
+
+    def generate(self, prompt: str, **_: object) -> str:
```

Ruff (0.14.4)

Unused method argument: `prompt`

(ARG002)